packages/documentation/plugins/remark/gatsby-remark-mdx-mermaid/remark-mermaid/src/index.js

```diff
@@ -0,0 +1,207 @@
+const fs = require('fs-extra');
```

Comment:
I had to modify the original version of mermaid to make this work with Gatsby. The original program didn't give us back the link and I needed to make sure the SVGs get generated in our static folder so we can use it on our pages.